app/MyApplicationLoader.scala

```diff
@@ -27,30 +32,37 @@ class MyComponents(context: Context)
   extends BuiltInComponentsFromContext(context)
     with AhcWSComponents
     with CSRFComponents
+    with HttpConfigurationComponents
```

Comment:
I updated play google auth to the latest that uses 2.6, and now it generates the anti forgery token from the session id rather than putting a random token in there just for the purpose (to allow concurrent auth requests).   https://github.com/guardian/play-googleauth/pull/52
It needs the httpConfiguration to generate and check the token.